### PR TITLE
No task request update

### PR DESCRIPTION
**Pull request by lgrzybowski**
Added new endpoints into repo.

### CODE REVIEW

File: lib/bwapi/client/pricing.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module BWAPI
+  class Client
+    module Pricing
+      def get_pricing(opts, params)
```
Comment (benSlaughter) [inline lib/bwapi/client/pricing.rb]: Do you need: `, opts = {}, params = {}`?

Comment (lgrzybowski) [reply lib/bwapi/client/pricing.rb]: a right :)

**Comment by lgrzybowski** [comment]
@benSlaughter you can take a look on it now :) Hows going mate ? I hope you are fine :)

**Comment by benSlaughter** [comment]
LGTM 👍  I can't see what the rubocop failures are 😆 

It's going very well, I am still getting so much information thrown at me.
boom

**Comment by lgrzybowski** [comment]
Looks like I need to update robocop to latest version... 
But I wont do this in this PR as it would be a massive change. 
Im happy that you are fine there @benSlaughter I hope to see you soon!!

**Pull request merged**